packages/terra-form-field/src/Field.jsx

```diff
@@ -100,11 +111,25 @@ const Field = (props, { intl }) => {
     isLabelHidden,
     label,
     labelAttrs,
+    maxWidth,
     required,
     showOptional,
+    style,
     ...customProps
   } = props;
 
+  let customStyles;
+
+  if (props.style) {
```

Comment:
Each occurrence of `props.style` can be simplified to `style` because it was deconstructed from props above.